marlowe-chain-sync/secrets/postgres_password

```diff
@@ -0,0 +1 @@
+9kZ@o7j4OkMC
```

Comment:
In the long term, we should probably avoid even default secrets in the commit.

Reply:
Yes, though this isn't sensitive info. Given that, it would probably be better if the `postgres` user for the dev database not to have a password at all.

Reply:
I'm just concerned about the case where someone blindly deploys the chain sync without changing the default password.